packages/sakuli-legacy/src/context/sahi/relations/relations-api.interface.ts

```diff
@@ -3,16 +3,122 @@ import { SahiElementQueryOrWebElement } from "../sahi-element.interface";
 import { SahiRelation } from "./sahi-relation.interface";
 
 export interface RelationApi extends ParentApi {
+  /**
+   * Create a relation that specifies elements inside an anchor element.
+   *
+   * @example
+   * <br />Assuming this HTML-snippet:
+   *
+   * ```html
+   * <b>something</b>
+   * <div id="div1">
+   *   <b>something</b>
+   * </div>
+   * ```
+   *
+   * The following expression ...
+   * ```typescript
+   * await _bold("something", _in(_byId("div1")))
+   * ```
+   * ... would return the second bold element.
+   * @param anchor
+   */
   _in(anchor: SahiElementQueryOrWebElement): SahiRelation;
+
+  /**
+   * Create a relation that sorts elements by how near they are to an anchor element.
+   *
+   * @param anchor
+   */
   _near(anchor: SahiElementQueryOrWebElement): SahiRelation;
+
+  /**
+   * Create a relation that specifies elements that are geometrically under an anchor element.
+   *
+   * **Note**: The specified elements must be intersected by the same vertical line. E.g.:
+   *
+   * ```
+   * +----+
+   * | 1  |
+   * +----+
+   *          +----+
+   *          | 2  |
+   *          +----+
+   *      +----+
+   *      | 3  |
+   *      +----+
+   * ```
+   * In this sense element `3` is  "under" element `2` and element `3` is also "under" element `1`, but element `2` is not "under" element `1`.
+   *
+   * **Note**: Reference for the relation is the geometrical center of each element.
+   * That means that elements, that are part of the relation, can overlap with the anchor element.
```

Comment:
Can you give me an example for this case? I don't understand what this means.

Reply:
The boxes of the elements (defined by the interface (https://github.com/sakuli/sakuli/blob/develop/packages/sakuli-legacy/src/context/sahi/relations/edges.function.ts)) can have overlapping borders.